### DISCUSSION TITLE
Zebra

### DISCUSSION THREAD

**Discussion by cedricchangy** [gopro/labs#251] [Feature Requests]
Hi, that could  be awsome if we could get zebra in the hero 11, added to logb it could be a great feature !

**Comment by andrerav** [comment]
That would be a nice feature indeed. I'm currently using HIST and EXPS to get an idea about the current exposure, and that works quite well. Zebra would be even better, but I'm not expecting that to be a simple hack.

**Comment by dnewman-gpsw** [comment]
There is a CPU bandwidth issue for a hack like this. The onscreen guides or the level control, both render into the LCD Preview buffer, but if you draw too much they can flicker.  For Zebra to work it needs to be about to read pixel values and potential render in to the whole preview buffer, preferably without flicker (missing frames.) If it was included as a mainline feature, a dedicated hardware resource could be used (like the GPU), but Labs is careful only to use low priority free CPU cycles to maintain the highest camera stability.  Worth trying though.

**Reply by dnewman-gpsw** [reply]
I quickly tried it, no good as impacted camera capture, it would need to be a deeper hack.
